src/scripts/main.js

```diff
@@ -1,3 +1,306 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+const sortedCols = {};
+let cellEditing = null;
+let cellInput = null;
+let initialCellText = '';
+let selectedRow;
+
+const getSalary = (salary) => {
+  return salary.innerText.slice(1).split(',').join('');
+};
+
+const sortByText = (rowsArray, index, inverseSort) => {
+  if (inverseSort === true) {
+    rowsArray.sort((a, b) => -a.cells[index].innerText
+      .localeCompare(b.cells[index].innerText));
+  } else {
+    rowsArray.sort((a, b) => a.cells[index].innerText
+      .localeCompare(b.cells[index].innerText));
```

Comment:
You write `.cells[index].innerText.localeCompare(b.cells[index].innerText))` not once. Create a method for it.